backend/scripts/test_litellm_connection.py

```
@@ -0,0 +1,184 @@
+LiteLLM 连接测试脚本
+用于诊断和测试 LiteLLM 代理连接问题
+"""
+
+import asyncio
+import sys
+from pathlib import Path
+
+# 添加项目根目录到Python路径
+sys.path.insert(0, str(Path(__file__).parent.parent))
+
+import httpx
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)